rotkehlchen/exchanges/htx.py

```diff
@@ -91,9 +126,10 @@ def _sign_request(self, url: str, params: dict[str, Any]) -> dict[str, Any]:
         new_params['Signature'] = base64.b64encode(dig).decode()
         return new_params
 
-    def _query(self, absolute_path: str) -> Any:
+    def _query(self, absolute_path: str, options: dict[str, Any] | None = None) -> Any:
         url = f'https://api.huobi.pro{absolute_path}'
-        signed_payload = self._sign_request(url=url, params={})
+        params = options or {}
+        signed_payload = self._sign_request(url=url, params=params)
```

Comment:
```suggestion
        signed_payload = self._sign_request(url=url, params=options if otions is not None else {})
```

Reply:
Changed it into a single line but ruff rule refactoring it to `options or {}`
https://docs.astral.sh/ruff/rules/if-exp-instead-of-or-operator/